addons/website_sale/static/src/scss/website_sale.scss

```
@@ -814,6 +814,44 @@ a.no-decoration {
     }
 }
 
-.accordion-button:not(.collapsed).bg-transparent::after {
-    background-image: escape-svg($accordion-button-icon);
-}
\ No newline at end of file
+.oe_website_sale {
+    // In website_sale the --accordion-active-bg is transparent or inherit
+    // which renders the wrong color on the chevron if the website background
+    // color is customized by the user.
+    .accordion-button:not(.collapsed)::after {
+        background-image: escape-svg($accordion-button-icon);
+    }
+
+    #product_accordion {
+        --accordion-active-bg: inherit;
+        --accordion-active-color: var(--accordion-btn-color);
+        --accordion-btn-padding-x : 0;
+        --accordion-body-padding-x: var(--accordion-btn-padding-x);
```

Why not simply `0`? does it change contextually somehow? 🤔 
```suggestion
        --accordion-body-padding-x: 0;
```